src/temgymbasic/components.py

```diff
@@ -240,15 +250,16 @@ class ParallelBeam(Source):
     def __init__(
         self,
         z: float,
+        wavelength: float,
         radius: float = None,
```

Comment:
`radius` should be required here (this is a bug on current `main`!), and also before `wavelength` if that becomes default